docs/xdr/features/collect/integrations/endpoint/vmware/vmware_vcenter.md

```diff
@@ -27,3 +27,45 @@ Go to the [intake page](https://app.sekoia.io/operations/intakes) and create a n
 ## Forward logs to Sekoia.io
 
 Please consult the [Syslog Forwarding](../../../../ingestion_methods/sekoiaio_forwarder/) documentation to forward these logs to Sekoia.io.
+
+Create a new configuration file:
+
+```
+sudo vim ./extended_conf/11-vcenter.conf
+```
+
+with the following template:
+
+```
+$DefaultNetstreamDriverCAFile /etc/rsyslog.d/Sekoia-io-intake.pem
+input(type="imtcp" port="20513" ruleset="remote20513")
```

Comment:
Maybe define a variable here instead of using a random port

Reply:
I put a placeholder instead